dashboard/app/views/admin_search/search_for_teachers.html.haml

```diff
@@ -47,7 +46,7 @@
     displayed.
     %br
 
-  #{link_to 'Download CSV', {:format => :csv}}
+  #{link_to 'Download CSV', {:format => :csv}} (commas and newlines have been replaced by spaces)
```

Comment:
The comma replacement happens in the csv.erb. Given that this is a comment to the user, it is accurate enough. Adding a comment comment for the developer.